forte/data/data_store.py

```diff
@@ -10,7 +10,7 @@
 class DataStore(BaseStore):
     # pylint: disable=pointless-string-statement
 
-    def __init__(self):
+    def __init__(self, onto_file_path=None):
         r"""An implementation from the dataframe-like data store object.
```

Comment:
This is not really "dataframe-like", we should say explicitly that this is an implementation that mainly use primitive types.

Reply:
yes, make sense